### PR TITLE
[IMP] *: remove global rules

### PR DESCRIPTION
**Pull request by mart-e**
and replace them with 3 rules for each role

Defining ir.model.access for records accessible for public/portal users should be public. Too often, rules were defined with `...model,,1,...` with "all = employees" in mind. Having access for portal/public is not forbidden but should be explicit.

This is the followup of task 3230280 to remove access-for-all unnecessary rules.
Transition to a system where global ACL no longer exists.

```
^([\w_]*)_all\,(.*)\,\,1,(\d),(\d),(\d)
$1_public,$2,base.group_public,1,$3,$4,$5
$1_portal,$2,base.group_portal,1,$3,$4,$5
$1_employee,$2,base.group_user,1,$3,$4,$5

^([\w_]*)\,(.*)\,\,1,(\d),(\d),(\d)
$1_public,$2,base.group_public,1,$3,$4,$5
$1_portal,$2,base.group_portal,1,$3,$4,$5
$1_employee,$2,base.group_user,1,$3,$4,$5
```

Task-id 3230280

### CODE REVIEW

File: odoo/modules/loading.py [hunk 1]
```diff
@@ -269,6 +269,21 @@ def load_module_graph(env, graph, status=None, perform_checks=True,
                         lines.append(f"{module_name}.access_{xmlid},access_{xmlid},{module_name}.model_{xmlid},base.group_user,1,0,0,0")
                     _logger.warning('\n'.join(lines))
```
[the remark this answers is not in the document]
Comment (gdgellatly) [reply odoo/modules/loading.py]: I quite like that line. From a development perspective, it is a touch lazy but I just don't think about security in anything new. Install it, cut and paste all the lines and edit groups and rights.

File: odoo/addons/base/security/ir.model.access.csv [hunk 2]
```diff
@@ -66,7 +80,9 @@
 "access_res_partner_industry_group_system","res_partner_industry group_system","model_res_partner_industry","group_system",1,1,1,1
 "access_res_partner_title_group_user","res_partner_title group_partner_manager","model_res_partner_title","group_partner_manager",1,1,1,1
 "access_res_partner_title_group_partner_manager","res_partner_title group_user","model_res_partner_title","group_user",1,0,0,0
-"access_res_users_all","res_users all","model_res_users",,1,0,0,0
+"access_res_users_all"_public,"res_users all","model_res_users",base.group_public,1,0,0,0
```
Comment (saeed-raeisi) [inline odoo/addons/base/security/ir.model.access.csv:83]: "access_res_users_all"_public,   -> "access_res_users_all_public"

**Comment by mart-e** [comment]
@robodoo r+ rebase-ff

**Comment by tde-banana-odoo** [comment]
Y U BREAK ZE FRIZE ?

**Comment by mart-e** [comment]
tumblr_n2pzwz0ueV1rrkahjo5_250

**Pull request closed without merging**